sdk/scheduler/src/main/java/com/mesosphere/sdk/config/validate/UserCannotChange.java

```diff
@@ -20,8 +20,22 @@
  * change as "nobody" will not have access to "root"-owned files in persistent volumes.
  */
 public class UserCannotChange implements ConfigValidator<ServiceSpec> {
```

Comment:
iiuc, previously we were validating the user change and throwing an error? (trying to understand what was the point of the error if we are using the new value for user anyway).

Reply:
We threw and error (which gets emitted as a log.warn) but the service continued on with the new values that *weren't* persisted to ZK. This might have been an edge case that was missed up till now.